src/go/rpk/pkg/cli/cmd/topic/consume.go

```diff
@@ -114,6 +119,12 @@ func NewConsumeCommand(client func() (sarama.Client, error)) *cobra.Command {
 		true,
 		"Pretty-print the consumed messages.",
 	)
+	cmd.Flags().BoolVar(
+		&metaOnly,
+		"meta",
```

Comment:
How about naming this `meta-only`? Even though users can always read the command's help text, I think it would be more explicit.

Reply:
I am good with `--meta-only`